src/remediations/fifi.js

```diff
@@ -43,6 +50,149 @@ exports.checkSmartManagement = async function (remediation, smart_management) {
     return _.some(systemsProfiles, system => system.system_profile.is_marketplace === true);
 };
 
+exports.sortSystems = function (systems, column = 'system_name', asc = true) {
+    return _.orderBy(systems, column, (asc) ? 'asc' : 'desc');
+};
+
+function createDispatcherRunsFilter (playbook_run_id = null) {
+    // Based on the qs library used in dispatcher connector, define filter in this format
+    const runsFilter = {filter: {}};
+    runsFilter.filter.service = SERVICE;
+
+    if (playbook_run_id) {
+        runsFilter.filter.labels = {'playbook-run': playbook_run_id};
+    }
+
+    return runsFilter;
+}
+
+function createDispatcherRunHostsFilter (playbook_run_id, host_id = null) {
+    const runHostsFilter = {filter: {run: {}}};
+    runHostsFilter.filter.run.service = SERVICE;
+    runHostsFilter.filter.run.labels = {'playbook-run': playbook_run_id};
+
+    if (host_id) {
+        runHostsFilter.filter.run.id = host_id;
+    }
+
+    return runHostsFilter;
+}
+
+function findRunStatus (run) {
+    if (run.count_failure > 0 || run.count_timeout > 0) {
+        return FAILURE;
+    } else if (run.count_running > 0 && run.count_timeout === 0 && run.count_failure === 0) {
+        return RUNNING;
+    } else if (run.success > 0 && run.timeout === 0) {
+        return SUCCESS;
+    }
+}
+
+function formatRHCRuns (rhcRuns) {
+    rhcRuns.system_count = rhcRuns.meta.count;
+
+    // Assign each status count
+    RHCSTATUSES.forEach(status => {
+        rhcRuns[`count_${status}`] = _.size(_.filter(rhcRuns.data, run => run.status === status));
+    });
+
+    // Status of executor
+    rhcRuns.status = findRunStatus(rhcRuns);
+}
+
+exports.formatRunHosts = function (rhcRunHosts) {
+    return _.map(rhcRunHosts.data, host => ({
+        system_id: host.id,
+        system_name: 'localhost',
```

Comment:
While technically this is correct as `localhost` is the hostname we use with RHC, I think this will get confusing to the customers if they see a paginated table where all entries are "localhost".

Reply:
This makes sense.  I'll swap it to something like the "host.id" instead

Reply:
@jharting once the inventory "link" PR is in I'll swap the 'system_name' to be the inventory host_id returned in links.  I'll make a subsiquent PR for that change.